src/main/java/lotto/GameController.java

```diff
@@ -0,0 +1,129 @@
+package lotto;
+
+import lotto.domain.lotto.Lotto;
+import lotto.domain.lotto.LottoAmount;
+import lotto.domain.lotto.LottoNumber;
+import lotto.domain.lotto.WinningLotto;
+import lotto.domain.rank.Rank;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import lotto.exception.LottoMoneyDivideException;
+import lotto.exception.LottoMoneyLessException;
+import lotto.exception.LottoNumNotNumException;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+import static lotto.view.OutputView.*;
+
+public class GameController {
+
+    private static final int TICKET_PRICE = 1000;
+    private static final int PERCENTAGE = 100;
+
+    private static LottoAmount lottoAmount;
+    private static List<Integer> lotto = new ArrayList<>();
+    private static List<Lotto> lottoList;
+    private static WinningLotto winningResult;
+    private static Map<Rank, Integer> result;
+
+    public GameController() {
+    }
+
+    public void start() {
+        int ticketCnt = amountToCnt();
+        printTicketCnt(ticketCnt);
+
+        lottoList = makeLottoList(ticketCnt);
+        winningResult = validateNums();
+
+        printResult(winningResult, ticketCnt);
+    }
+
+    public int amountToCnt(){
+        while (true) {
+            try {
+                lottoAmount = new LottoAmount(InputView.getInputAmount());
+                return lottoAmount.amountChangeToLottoCnt();
+            } catch (LottoNumNotNumException e) {
+                System.out.println(e.getMessage());
+            } catch (LottoMoneyLessException e) {
+                System.out.println(e.getMessage());
+            } catch (LottoMoneyDivideException e) {
+                System.out.println(e.getMessage());
+            }
```

Comment:
(이렇게 변경하면 좋을 것 같아요!)
```suggestion
            } catch (LottoNumNotNumException  | LottoMoneyLessException | LottoMoneyDivideException e) {
                System.out.println(e.getMessage());
            }
           
```
https://dololak.tistory.com/61